### PR TITLE
[feature][trino-python-client] add new trino python client && support trino set session.

### PR DESCRIPTION
**Pull request by zhuzhengjun01**
## What changes were proposed in this pull request?
Added a new trino python client to support the execution of commands such as trino set sessions through the python client

## How was this patch tested?

- (Please explain how this patch was tested. Ex: unit tests, manual tests)
it is test by manual tests, however,this code if forked from hue ,  and most of the code come from hue original

### CODE REVIEW

File: ~ [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pick 8e73ecf23f [feature][trino-python-client] add new trino python client && support trino set session
```
Comment (amitsrivastava) [inline ~:1]: I think this file came by a mistake. Please check.

**Review by amitsrivastava — COMMENTED** [review]
@zhuzhengjun01 thanks for the contribution. Can you explain the need for a new trino client, given that a trino client was recently added to Hue?

**Comment by zhuzhengjun01** [comment]
> @zhuzhengjun01 thanks for the contribution. Can you explain the need for a new trino client, given that a trino client was recently added to Hue?



> @zhuzhengjun01 thanks for the contribution. Can you explain the need for a new trino client, given that a trino client was recently added to Hue?

Thanks for your reply~ 
The original trino connector does not support set session and multi-sql execution. In addition, if you connect trino through jdbc, there will be a memory leak problem. In order to solve this problem, it was changed to python client, which is different from the original one.

issue: https://github.com/cloudera/hue/issues/3223

File: ~ (hunk 1 above)
Comment (zhuzhengjun01) [reply ~:1]: > I think this file came by a mistake. Please check.

Sry  I will remove this file

Comment (zhuzhengjun01) [reply ~]: This file has been removed, may caused by git rebase

**Pull request closed without merging**